src/middlewared/middlewared/plugins/service_/services/pseudo/misc.py

```diff
@@ -199,7 +199,7 @@ async def start(self):
 class SysconsService(SimpleService):
```

Comment:
Can we get rid of this pseudo-service and just do the job directly in the plugin?

Reply:
Sure!